packages/lib-classifier/src/plugins/tasks/experimental/highlighter/models/HighlighterAnnotation.js

```diff
@@ -0,0 +1,29 @@
+import { types } from 'mobx-state-tree'
+
+import Annotation from '@plugins/tasks/models/Annotation'
+import HighlighterLabel from './HighlighterLabel'
+
+const Highlight = types.model('Highlight', {
+  start: types.optional(types.number, 0),
+  end: types.optional(types.number, 0),
+  text: types.optional(types.string, ''),
+  labelInformation: types.optional(HighlighterLabel, { color: '', label: '' })
+})
+
+const Highlighter = types.model('Highlighter', {
+  taskType: types.literal('highlighter'),
+  value: types.optional(types.array(Highlight), [])
+})
+  .views(self => ({
+    get isComplete () {
+      return self.value.length > 0
```

Comment:
Could a volunteer be shown a page that doesn't have any text labels? If so, this might catch them out by requiring at least one label on each subject.

Reply:
The highlighter task isn't required (something to revisit, it prob should have the option to be required), so as is a user can not highlight anything and will be allowed to proceed (Next/Done).

Here's r[elated task model code](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/plugins/tasks/models/Task.js#L36-L38).
Here's a link to the a [highlighter task in the Project Builder](https://master.pfe-preview.zooniverse.org/lab/1952/workflows/3538).